common/shlibs

```diff
@@ -4035,3 +4035,4 @@ libstemmer.so.2 libstemmer-2.1.0_1
 libdecor-0.so.0 libdecor-0.1.0_1
 libavif.so.13 libavif-0.9.3_1
 libkdumpfile.so.8 libkdumpfile-0.4.0_1
+libgpsdpacket.so.29 gpsd-3.23.1_3
```

Comment:
This warning is harmless, so please leave it out for now. It's necessary only if some other package depends on it.

Reply:
Moving the .so is fine :)

If you want, `vmove "usr/lib/*.so"` should also work.